example/index.js

```diff
@@ -70,6 +70,22 @@ db.installUsersBehavior().then(function () {
     }
   })
 
+  server.route({
+    method: 'GET',
+    path: '/default/css/default.css',
+    handler: function (request, reply) {
+      reply.file(path.join(__dirname, '..', 'public', 'client.css'))
+    }
+  })
+
+  server.route({
+    method: 'GET',
+    path: '/default/js/default.js',
+    handler: function (request, reply) {
+      reply.file(path.join(__dirname, '..', 'public', 'client.js'))
+    }
+  })
```

Comment:
instead of the three `reply.file` statements, you can do 

```js
server.route({
  method: 'GET',
  path: '/{p*}',
  handler: {
    directory: {
      path: path.join(__dirname, '..', 'public'),
      listing: false,
      index: true
    }
  }
})
```

and adjust the paths of `<script src="/default/js/default.js"></script>` and `<link rel="stylesheet" href="/default/css/default.css">` to `./client.js` and `./client.css`